lib/maintainers.nix

```diff
@@ -12,6 +12,7 @@
   abbradar = "Nikolay Amiantov <ab@fmap.me>";
   aboseley = "Adam Boseley <adam.boseley@gmail.com>";
   adev = "Adrien Devresse <adev@adev.name>";
+  adjective-object = "Maxwell Huang-Hobbs <mhuan13@gmail.com>";
```

Comment:
Please make it the exact same as your github handle; this is to make it easier to ping you on github if necessary